lib/galaxy/webapps/galaxy/controllers/workflow.py

```diff
@@ -580,32 +556,6 @@ def for_direct_import(self, trans, id):
         stored = self.get_stored_workflow(trans, id, check_ownership=False, check_accessible=True)
         return self._workflow_to_dict(trans, stored)
 
-    @web.json_pretty
-    def export_to_file(self, trans, id):
```

Comment:
This method seems to still be used in this file at line 246.

Reply:
Thanks, that's what I get by just looking at web.* decorators ...